tests/kubeless-test-client/README.md

```diff
@@ -1,9 +1,7 @@
 The `kubeless-test-client` contains the following:
-* A `Dockerfile` for the image used in Kyma kubeless tests
-* A go file named test-kubeless.go, which executes the tests for the kubeless chart
-
-Use the following command to build image:
-
-```bash
-$ make build-release-image RELEASE_VERSION=1.6.0
-```
+* A `Dockerfile` for the image used in Kyma Kubeless tests
+* A Go file named `test-kubeless.go`, which executes the tests for the Kubeless chart
+* A namespace YAML, which specifies the `kubeless-test` namespace
```

Comment:
A Namespace [configuration file](link to the file) which defines the `kubeless-test` Namespace. 


----
Namespaces are Kubernetes resources - capitalize pls. See https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/style-and-standards.md#capitalized-terms
for details.

Reply:
Fixed